persist/src/main/resources/db/migration/V15__AddChildProfilePicture.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE children ADD COLUMN profile_picture VARCHAR(500) DEFAULT null;
+ALTER TABLE contacts
+RENAME COLUMN profilepicture TO profile_picture;
```

Comment:
```suggestion
ALTER TABLE children 
ADD COLUMN profile_picture VARCHAR(500) DEFAULT null;

ALTER TABLE contacts
RENAME COLUMN profilePicture TO profile_picture;

```

Few things: 
1. formatting, put things on their own lines.
2. space between separate statements. 
3. profilepicture != profilePicture
4. because this isn't deployed to prod, instead of doing the rename column you could just fix the first migration and/or combine these, you'd just have to clear your local DB. 
5. newline at end of file